packages/compile-common/package.json

```diff
@@ -4,11 +4,15 @@
   "description": "Common compiler integration support infrastructure for Truffle",
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
+  "homepage": "https://github.com/trufflesuite/truffle/tree/master/packages/compile-common#readme",
```

Comment:
compile-common doesn't have a readme

Reply:
OK, so are you saying I should remove the anchor here (and in similar cases) for now?

Reply:
Removing the anchor creates a workable (non breaking) URL but I'm not sure it is the right thing to do. I would prefer a minimal readme. Can we vote on it?

Reply:
OK, so you were just noting the missing readmes to note them, not to imply that any change should be made at that point.

I think adding readmes is outside the scope of this PR, and would be better done in a different one.  If there's not any need to change the URLs I'm going to merge this as-is.